scripts/windows/test/Run-E2ETest.ps1

```diff
@@ -1087,21 +1087,40 @@ Function RunTransparentGatewayTest
     $testCommand = AppendInstallationOption($testCommand)
     Invoke-Expression $testCommand | Out-Host
```

Comment:
please check result of this invoke-expression as well using sth like "$testExitCode = $LastExitCode".

Reply:
Will do, added!